src/app/components/room/room.service.js

```diff
@@ -15,6 +15,11 @@
       'FeedConnection', 'DataChannelService', 'ActionService', 'jhConfig',
       'ScreenShareService'];
 
+  /**
+   * Service containing all rooms
```

Comment:
It provides functions to enter a room, to publish a new feed in the room, to get the list of participants in a room and attach to their feeds... Everything is related to THE room, which is kind of a singleton entity, since you cannot operate in two rooms at the same time.

At some point (it was not there initially) we added a function to get the list of rooms (in plural) and even a `RoomsFactory` to represent the attributes of rooms, but the name `RoomService` was still good enough to stay (and rather funny). :wink: It's a service to perform operations on the room.

A service created to contain all the rooms (if such thing would exist) would be called `RoomsService` (note the plural).